components/climate/midea_ac.rst

```diff
@@ -0,0 +1,92 @@
+Midea Air Conditioner
+=====================
+
+.. seo::
+    :description: Instructions for setting up a Midea climate device
+    :image: air-conditioner.png
+
+The ``midea_ac`` component creates a Midea air conditioner climate device.
+
+This component requires a :doc:`/components/midea_dongle` to be configured.
+
+.. note::
+
+    This protocol also used by some vendors:
+
+        - `Electrolux <https://www.electrolux.ru/>`_
+        - `Qlima <https://www.qlima.com/>`_
+        - `Artel <https://www.artelgroup.com/>`_
+        - `Carrier <https://www.carrier.com/>`_
+        - `Comfee <http://www.comfee-russia.ru/>`_
+        - `Inventor <https://www.inventorairconditioner.com/>`_
+        - and maybe others
+
+    Example of hardware implementation is `Midea Open Dongle <https://github.com/dudanov/midea-open-dongle>`_ in free `KiCad <https://kicad-pcb.org>`_ format.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    # Disable logging over UART
+    logger:
+      baud_rate: 0
+
+    uart:
+      tx_pin: 1
+      rx_pin: 3
+      baud_rate: 9600
+
+    midea_dongle:
+
+    midea_ac:
+      name: "Midea AC"
+      outdoor_temperature:
+        name: "outdoor"
+      beeper: true
+      swing_horizontal: true
+      swing_both: true
+
+Configuration variables:
+------------------------
+
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **midea_dongle_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :doc:`/components/midea_dongle` if you want to use multiple devices.
+- **name** (**Required**, string): The name of the climate device.
+- **outdoor_temperature** (*Optional*): The information for the outdoor temperature
+  sensor.
+
+  - **name** (**Required**, string): The name of the sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **power_usage** (*Optional*): The information for the current power consumption
+  sensor.
+
+  - **name** (**Required**, string): The name of the sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **humidity_setpoint** (*Optional*): The information for the humidity indoor
+  sensor (experimental).
+
+  - **name** (**Required**, string): The name of the sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **beeper** (*Optional*, bool, default: **false**): Beeper feedback on command.
+- **swing_horizontal** (*Optional*, bool, default: **false**): Enable **swing horizontal** option.
+- **swing_both** (*Optional*, bool, default: **false**): Enable **swing both** option.
```

Comment:
This is not how we document defaults, see other components for examples.
Usually a statement at the end like: "Defaults to ``false``."

Reply:
Done.